test/tools/libtesteth/TestHelper.cpp

```diff
@@ -452,6 +456,31 @@ void executeTests(const string& _name, const string& _testPathAppendix, const st
 	}
 }
 
+void addClientInfo(json_spirit::mValue& _v, std::string const& _testSource)
+{
+	for (auto& i: _v.get_obj())
+	{
+		json_spirit::mObject& o = i.second.get_obj();
+		json_spirit::mObject clientinfo;
+
+		string head = executeCmd("git rev-parse HEAD");
```

Comment:
Yeah, there should be a nicer way, I don't like launching a separate process for this, even less for doing it in the loop (launching a process for each file to get the same string)